CMakeLists.txt

```diff
@@ -140,6 +140,8 @@ if(MSVC)
             if(NOT HAVE_PTHREAD_H)
                 message(FATAL_ERROR "Could not find pthreads.")
             endif()
+            unset(PThreads_LIBRARIES)
+            unset(PThreads_INCLUDE_DIRS)
```

Comment:
What works for me is
```
find_path(PThreads_INCLUDE_DIRS NAMES pthread.h )
find_library(PThreads_LIBRARIES NAMES libwinpthread.dll pthreads libpthreads pthread)
```
instead of line 138 to 144 (i.e. without the `HAVE_PTHREAD_H` check, which I think is made obsolete by the `find_path`).

Reply:
Please make a code suggestion.